simtools/db_handler.py

```diff
@@ -182,21 +180,26 @@ def getModelParameters(
 
     def exportFileDB(self, dbName, dest, fileName):
         """
-        Get a file from the DB and write it to disk.
+        Get file from the DB and write to disk.
 
         Parameters
         ----------
         dbName: str
             Name of the DB to search in.
         dest: str or Path
             Location where to write the file to.
-        fileName: str
+        file_name: str
             Name of the file to get.
+        Returns
+        -------
+        GridOut
+            :class:`~gridfs.grid_file.GridOut` or ``None`` if file not found
```

Comment:
You are not actually returning GridOut, you are returning the file ID. The thing is, I don't think the file ID is so useful as a return value (nor the GridOut). If you wrote a file, that's it, no additional info is necessary in the return value. If the file was not found you are raising an error (I think we should add that info about the raising of the error to the docstring).

Reply:
Well spotted, it is the ID that is returning. I do think it is good to return the id, at least for the case where we test the insert function. probably the user will not care for the id, in which case he/she can just ignore it and not even assign a variable for keeping it, no problem. I will add the warning to the docstring